src/routes/login/+page.server.test.ts

```
@@ -0,0 +1,242 @@
+			const mockCookies = {
+				set: vi.fn(),
+				get: vi.fn(),
+				delete: vi.fn(),
+				getAll: vi.fn(),
+				serialize: vi.fn()
+			};
+
+			const result = await actions.default({
+				request: mockRequest,
+				cookies: mockCookies
+			} as any);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)